cloudferrylib/base/resource.py

```diff
@@ -36,8 +41,18 @@ def save(self):
     def restore(self):
         pass
 
-    def wait_for_status(self, id_obj, status, limit_retry=60):
-        pass
+    def wait_for_status(self, res_id, get_status, wait_status, limit_retry=60, warn_only=False):
```

Comment:
> find . -name '.py' -not -wholename './.venv/' | xargs .venv/bin/flake8 --max-line-length 160 || true

Not funny. Max line length is [79 according to pep8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length)